proto/api.proto

```diff
@@ -60,6 +60,11 @@ message Player {
 	SimDatabase database = 18;
 	HealingModel healing_model = 19;
 
+	// Stop Attack Macro Info
+	bool IsUsingCrusaderStrikeStopAttack = 49;
+	bool IsUsingJudgementStopAttack = 50;
+	bool IsUsingDivineStormStopAttack = 51;
```

Comment:
These options should be in the `paladin.proto` options message since they're class/spec-specific